models/src_shopify.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 
 sources:
-  - name: shopify
+  - name: shopify # only if you're using a singular connector
```

Comment:
Should we keep this comment? The customer won't be able to adjust it and I worry this would cause some confusion. What are your thoughts?

Reply:
hm maybe i can move it to line 2? i feel like it should be noted that this is only used if you have one source connector

Reply:
Gotchya, I think it is definitely still relevant to include here then. Maybe we can adjust the wording a bit. Currently it reads like it is on me (the user) to set something. What about, "This source will only be used if you are using a single source connection. If multiple sources are being unioned...etc"

Reply:
added